lib/ecto/changeset.ex

```diff
@@ -568,7 +570,10 @@ defmodule Ecto.Changeset do
             end
 
           :error ->
-            :invalid
+            {:invalid, message: "is invalid"}
```

Comment:
Since we do have a default message, I think we can go with:

```suggestion
            {:invalid, []}
```